docs/zephyr-integration.md

```diff
@@ -0,0 +1,493 @@
+# Zephyr Test Management Integration
+
+## Overview
+
+The Zephyr MCP integration provides AI assistants with comprehensive test management capabilities through the SmartBear MCP (Model Context Protocol) server. This integration enables seamless test case creation, test planning, execution tracking, and issue coverage analysis through conversational AI interfaces.
+
+### Key Benefits
+
+- **Automated Test Management**: Create and manage test cases, plans, and executions through natural language
+- **Issue Traceability**: Analyze test coverage for Jira issues and requirements
+- **Streamlined Workflows**: Execute complex test management tasks without manual UI navigation
+- **Comprehensive API Access**: Full access to Zephyr Cloud functionality through standardized MCP tools
+
+## Setup
+
+### Environment Variables
+
+The following environment variables configure the Zephyr integration:
+
+#### Required
+
+- **`ZEPHYR_ACCESS_TOKEN`** (required): JWT access token for Zephyr Cloud API authentication
+  - Obtain from Zephyr Cloud: Account Settings → API Tokens
+  - Format: Standard JWT token (3 base64url-encoded parts separated by dots)
+  - Example: `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...`
+
+#### Optional
+
+- **`ZEPHYR_PROJECT_KEY`** (optional): Default project scope for operations
```

Comment:
I think this can cause confusion, what if the user has a default project but we ask MCP to create in another project then it does not care about it... and later if we are on the context of this other project any other instruction will be on this other project, so I think this project_key as env variable can cause confusion